src/Spreadsheet.js

```diff
@@ -110,10 +110,11 @@ class Spreadsheet<CellType, Value> extends PureComponent<{|
     DataViewer,
     DataEditor,
     getValue,
-    getBindingsForCell,
-    formulaParser: new FormulaParser()
+    getBindingsForCell
   };
 
+  formulaParser = this.props.formulaParser || new FormulaParser();
```

Comment:
Of course and thank you for the quick response @iddan!

Is this what you had in mind or did I misunderstand?

Reply:
Yes, thank you